core/renderers/geras/renderer.js

```diff
@@ -35,116 +34,118 @@ const {Theme} = goog.requireType('Blockly.Theme');
-Renderer.prototype.init = function(theme, opt_rendererOverrides) {
-  Renderer.superClass_.init.call(this, theme, opt_rendererOverrides);
-  this.highlightConstants_ = this.makeHighlightConstants_();
-  this.highlightConstants_.init();
-};
+    /**
+     * The renderer's highlight constant provider.
+     * @type {HighlightConstantProvider}
+     * @private
+     */
+    this.highlightConstants_ = null;
+  }
 
-/**
- * @override
- */
-Renderer.prototype.refreshDom = function(svg, theme) {
-  Renderer.superClass_.refreshDom.call(this, svg, theme);
-  this.getHighlightConstants().init();
-};
+  /**
+   * Initialize the renderer.  Geras has a highlight provider in addition to
+   * the normal constant provider.
+   * @package
+   * @override
+   */
+  init(theme, opt_rendererOverrides) {
+    super.init(theme, opt_rendererOverrides);
+    this.highlightConstants_ = this.makeHighlightConstants_();
+    this.highlightConstants_.init();
+  }
 
-/**
- * @override
- */
-Renderer.prototype.makeConstants_ = function() {
-  return new ConstantProvider();
-};
+  /**
+   * @override
+   */
+  refreshDom(svg, theme) {
+    super.refreshDom(svg, theme);
+    this.getHighlightConstants().init();
+  }
 
-/**
- * Create a new instance of the renderer's render info object.
- * @param {!BlockSvg} block The block to measure.
- * @return {!RenderInfo} The render info object.
- * @protected
- * @override
- */
-Renderer.prototype.makeRenderInfo_ = function(block) {
-  return new RenderInfo(this, block);
-};
+  /**
+   * @override
+   */
+  makeConstants_() {
+    return new ConstantProvider();
+  }
 
-/**
- * Create a new instance of the renderer's drawer.
- * @param {!BlockSvg} block The block to render.
- * @param {!BaseRenderInfo} info An object containing all
- *   information needed to render this block.
- * @return {!Drawer} The drawer.
- * @protected
- * @override
- */
-Renderer.prototype.makeDrawer_ = function(block, info) {
-  return new Drawer(
-      block,
-      /** @type {!RenderInfo} */ (info));
-};
+  /**
+   * Create a new instance of the renderer's render info object.
+   * @param {!BlockSvg} block The block to measure.
+   * @return {!RenderInfo} The render info object.
+   * @protected
+   * @override
+   */
+  makeRenderInfo_(block) {
+    return new RenderInfo(this, block);
+  }
 
-/**
- * Create a new instance of a renderer path object.
- * @param {!SVGElement} root The root SVG element.
- * @param {!Theme.BlockStyle} style The style object to use for
- *     colouring.
- * @return {!PathObject} The renderer path object.
- * @package
- * @override
- */
-Renderer.prototype.makePathObject = function(root, style) {
-  return new PathObject(
-      root, style,
-      /** @type {!ConstantProvider} */ (this.getConstants()));
-};
+  /**
+   * Create a new instance of the renderer's drawer.
+   * @param {!BlockSvg} block The block to render.
+   * @param {!BaseRenderInfo} info An object containing all
+   *   information needed to render this block.
```

Comment:
The wrapping indentation seems a bit off. I believe it is 2 spaces when it should be 4.

Reply:
Fixed.